Code/GraphMol/RGroupDecomposition/RGroupDecompParams.cpp

```diff
@@ -54,11 +54,11 @@ unsigned int RGroupDecompositionParameters::autoGetLabels(const RWMol &core) {
 
 bool rgdAtomCompare(const MCSAtomCompareParameters &p, const ROMol &mol1,
                     unsigned int atom1, const ROMol &mol2, unsigned int atom2,
-                    void *userData) {
-  if (!MCSAtomCompareElements(p, mol1, atom1, mol2, atom2, nullptr)) {
+                    MCSCompareFunctionsData& cfd) {
+  if (!MCSAtomCompareElements(p, mol1, atom1, mol2, atom2, cfd)) {
```

Comment:
with the new structure for MCSCompareFunctionsData, the MCSAtomCompareElements functor can pass around dynamic data and custom functors can call it without a problem